resources/zib/zib-MultidisciplinaryTeamMeeting.xml

```diff
@@ -0,0 +1,239 @@
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-PractitionerRole" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactPerson" />
+      </type>
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.3" />
+        <comment value="HealthProfessional" />
+      </mapping>
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.15" />
+        <comment value="Contactperson" />
+      </mapping>
+    </element>
+    <element id="Encounter.participant.individual.extension">
+      <path value="Encounter.participant.individual.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="Encounter.participant.individual.extension:patient">
+      <path value="Encounter.participant.individual.extension" />
+      <sliceName value="patient" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Encounter-PatientParticipant" />
+      </type>
+    </element>
+    <element id="Encounter.participant.individual.extension:patient.value[x]">
+      <path value="Encounter.participant.individual.extension.value[x]" />
+      <short value="Patient" />
+      <definition value="The patient as participant of the patient consultation." />
+      <alias value="Patient" />
+      <mapping>
+        <identity value="zib-multidisciplinaryteammeeting-v1.0-2020EN" />
+        <map value="NL-CM:15.2.16" />
+        <comment value="Patient" />
+      </mapping>
+    </element>
+    <element id="Encounter.period">
+      <path value="Encounter.period" />
+      <comment value="'PatientConsultationDateTime' is mapped to both `period.start` and `period.end`. When all is known is a single value, both `.start` and `.end` should be populated with this value, otherwise the `period`, and thus the MultidisciplinaryTeamMeeting, is considered 'ongoing'. In FHIR R4, the Period datatype has an invariant that inconsistently says in text that Period.start shall be lower than Period.end, but actually supports lower than or equal in its expression, which reflects the original intent. This has been fixed for FHIR R5; the text now reflects the expression. &#xD;&#xA;&#xD;&#xA;When a system is able to populate both `period.start` and `period.end`, the value of `.start` is leading when converting FHIR data to a zib instance." />
```

Comment:
Should Period.start and Period.end also have quotes (`), or do we only use those if it concerns actual elements within the respective profile (such as `period.start` and `period.end` in this case)?

Reply:
Your comment is kind of hard to read because of markdown.

ElementDefinition.comment is markdown, so I used single quotes to represent a zib concept and backticks for fhir elements. But I guess I missed some.

Did this (and the correction) answer your question?